deploy: update node-driver-registrar to latest version

humblec opened the pull request:
This commit update the node driver registrar to the latest
available version.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec committed:
deploy: update node-driver-registrar to latest version

This commit update the node driver registrar to the latest
available version.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Madhu-1 commented on build.env:
```diff
@@ -52,7 +52,7 @@ CSI_ATTACHER_VERSION=v3.4.0
 CSI_PROVISIONER_VERSION=v3.1.0
 CSI_RESIZER_VERSION=v1.4.0
-CSI_NODE_DRIVER_REGISTRAR_VERSION=v2.4.0
+CSI_NODE_DRIVER_REGISTRAR_VERSION=v2.5.1
```
why update only the build.env, not the deployment yaml files?

humblec replied:
Huh.. missed to commit.. :( .updating

humblec committed:
deploy: update node-driver-registrar sidecar to latest

updated deployment yamls for the latest version.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec committed:
helm: updated node driver registrar to latest version

the helm chart template value has been updated to latest
version of node driver registrar container.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec commented:
/retest  ci/centos/mini-e2e-helm/k8s-1.23

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.23

nixpanic commented:
the `ci/retry/e2e` label only makes sense when all approvals are there, and there are no other PRs in the queue. Please do not add it when not required (yet).

nixpanic commented on deploy/rbd/kubernetes/csi-rbdplugin.yaml:
```diff
@@ -29,7 +29,7 @@ spec:
             privileged: true
             allowPrivilegeEscalation: true
-          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.4.0
+          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.5.1
```
this commit misses the update for the NFS-provisioner

humblec replied:
@nixpanic done.. thanks.. ptal.

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.21

humblec commented:
@nixpanic comments are addressed .ptal. thanks

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.21

humblec commented:
@mergifyio refresh

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.21

nixpanic commented:
@Mergifyio requeue

humblec commented:
Hmmm.. 2 days to get the PR in ,, still not !!

humblec commented:
@Mergifyio refresh

humblec commented:
@Mergifyio rebase
